src/engraving/dom/trill.cpp

```diff
@@ -295,7 +295,7 @@ PointF Trill::trillLinePos(const SLine* line, Grip grip, System** system)
 
     // Stop line before grace notes
     if (graceNoteSeg) {
-        const EngravingItem* preAppendedItem = graceNoteSeg->preAppendedItem(line->track2());
+        const EngravingItem* preAppendedItem = graceNoteSeg->preAppendedItem(static_cast<int>(line->track2()));
```

Comment:
I'd prefer changing the signatures of `preAppendedItem` and related methods to take track_idx_t

Reply:
OK, will do. It'll be quite a bit larger